Dockerfile

```diff
@@ -1,19 +1,17 @@
 # Stage1: Build the pvc-autoresizer binary
-FROM golang:1.20-bullseye as builder
+FROM golang:1.20 as builder
```

Comment:
I don't know why we are using "bullseye" image, but why did you change it?

Reply:
> why we are using "bullseye"

There are two images in golang:1.20, bullseye base and bookworm base. When updating from golang:1.19, we specified bullseye explicit for the base image to keep compatibility. 

> why did you change it?

Since pvc-autoresizer programs are static-linked and run in the scratch environment, the base image of the build environment is not an important factor. Therefore, the base image specification is removed to follow the template.

Reply:
Okay I understand that this change is a reasonable one, although not seems to be so much related to migration of kubebuilder. Thank you.